### PR TITLE
Use the standard do-not-edit comments

### PR DESCRIPTION
**Pull request by hajimehoshi**
This PR changes or adds the template comments in order to follow the standard rule: https://golang.org/s/generatedcode

@dmitshur PTAL

### CODE REVIEW

File: tmpl/conversions.tmpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //glow:keepspace
-// Glow automatically generated OpenGL binding: http://github.com/go-gl/glow
+// Code generated by glow (http://github.com/go-gl/glow). DO NOT EDIT.
```
Comment (dmitshur) [inline tmpl/conversions.tmpl]: Please update the URL scheme to https. Here, and below.

```suggestion
// Code generated by glow (https://github.com/go-gl/glow). DO NOT EDIT.
```

File: tmpl/debug.tmpl [hunk 2]
```diff
@@ -1,5 +1,5 @@
 //glow:keepspace
-// Glow automatically generated OpenGL binding: http://github.com/go-gl/glow
+// Code generated by glow (http://github.com/go-gl/glow). DO NOT EDIT.
```
Comment (dmitshur) [inline tmpl/debug.tmpl]:
```suggestion
// Code generated by glow (https://github.com/go-gl/glow). DO NOT EDIT.
```

File: tmpl/conversions.tmpl (hunk 1 above)
Comment (hajimehoshi) [reply tmpl/conversions.tmpl]: Done

File: tmpl/debug.tmpl (hunk 2 above)
Comment (hajimehoshi) [reply tmpl/debug.tmpl]: Done.

**Comment by hajimehoshi** [comment]
Hi, thank you for the quick review. I don't have right to merge this. Could you merge this?

**Pull request merged**